packages/nextjs/app/debug/page.tsx

```diff
@@ -0,0 +1,35 @@
+import DebugContractsComponents from "../../components/debug/debugContractsComponent";
+import type { Metadata, NextPage } from "next";
+
+export const metadata: Metadata = {
+  title: "Debug Contract",
```

Comment:
same question about this title, it was `Debug Contracts | Scaffold-ETH 2`

Reply:
Got it, thanks! But change it to `Debug Contracts`

Reply:
done